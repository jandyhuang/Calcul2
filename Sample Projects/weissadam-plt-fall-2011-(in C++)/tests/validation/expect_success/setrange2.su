function main[] returns int
{
 int b = -10;
 int e = 10;
 set a = {b ... e};
 return 0;
}
