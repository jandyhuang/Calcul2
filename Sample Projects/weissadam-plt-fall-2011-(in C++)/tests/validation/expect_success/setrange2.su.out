a set_seq
e int
b int
